pkgs/by-name/fl/fluent-bit/package.nix

```diff
@@ -1,81 +1,147 @@
 {
   lib,
-  stdenv,
-  fetchFromGitHub,
+  arrow-glib,
+  bison,
+  c-ares,
   cmake,
+  curl,
+  fetchFromGitHub,
   flex,
-  bison,
-  systemd,
-  postgresql,
-  openssl,
+  jemalloc,
+  libbacktrace,
+  libbpf,
+  libnghttp2,
   libyaml,
-  darwin,
+  luajit,
+  nix-update-script,
+  nixosTests,
+  openssl,
+  pkg-config,
+  postgresql,
+  rdkafka,
+  stdenv,
+  systemd,
+  versionCheckHook,
+  zlib,
+  zstd,
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation rec {
```

Comment:
We should keep `finalAttrs` here else `version` is not overridable. 

`rec` in `mkDerivation` is considered an anti-pattern